a10_octavia/tests/unit/controller/worker/tasks/test_network_tasks.py

```diff
@@ -0,0 +1,50 @@
+#    Copyright 2020, A10 Networks
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+# -*- coding: utf-8 -*-
+
+import mock
+
+from octavia.tests.common import constants as t_constants
+import octavia.tests.unit.base as base
+
+from a10_octavia.controller.worker.tasks import a10_network_tasks
+from a10_octavia.network import data_models
+from a10_octavia.network.data_models import Subport
+from a10_octavia.tests.common import a10constants as a10_test_constants
+
+SUBPORT = Subport(segmentation_id=a10_test_constants.SEGMENTATION_ID)
+
+
+class TestNetworkTasks(base.TestCase):
+
+    @mock.patch('a10_octavia.controller.worker.tasks.a10_network_tasks.BaseNetworkTask.network_driver')
+    def test_handle_delete_port_deltas(self, get_network_drivers_mock):
```

Comment:
Testing network tasks wasn't captured in the story that this work encompasses. Nor was the creation of this listed as a task on the story.

Please only work on things we identify within the stories we write otherwise we don't have a way of tracking work completed and the time it takes to complete such work.